addons/xeh/fnc_preInit.sqf

```diff
@@ -63,11 +63,11 @@ GVAR(EventsLowercase) = [];
         FUNC(Init) = compileFinal (_header + "(_this select 0) call CBA_fnc_initEvents; (_this select 0) call CBA_fnc_init");
     } else {
         if (_x isEqualTo "HitPart") then {
-            FUNC(HitPart) = compileFinal (_header + format ['{call _x; nil} count ((_this select 0 select 0) getVariable QGVAR(%1))', _x]);
+            FUNC(HitPart) = compileFinal (_header + format ['{call _x} forEach ((_this select 0 select 0) getVariable QGVAR(%1))', _x]);
         } else {
             missionNamespace setVariable [
                 format [QFUNC(%1), _x],
-                compileFinal (_header + format ['{call _x; nil} count ((_this select 0) getVariable QGVAR(%1))', _x])
+                compileFinal (_header + format ['{call _x} forEach ((_this select 0) getVariable QGVAR(%1))', _x])
```

Comment:
@commy2 - Is this safe, or do we have to worry about the return type changing? 
forEach returns last run
instead of `nil } count` always returns 0

Reply:
No, this is fine.